File: sharedse/src/main/java/com/ustadmobile/port/sharedse/impl/http/UmHttpCallSe.java
package com.ustadmobile.port.sharedse.impl.http;

import com.ustadmobile.core.impl.http.UmHttpCall;

import okhttp3.Call;

/**
 * Simple wrapper to map to OK HTTP library
 */

public class UmHttpCallSe extends UmHttpCall{

    private Call call;

    /**
     *
     * @param call the OK Http Call object
     */
    public UmHttpCallSe(Call call) {
        this.call = call;
    }

    @Override
    public void cancel() {
        call.cancel();
    }
}

File: sharedse/src/main/java/com/ustadmobile/port/sharedse/impl/http/UmHttpCallSe.kt
package com.ustadmobile.port.sharedse.impl.http

import com.ustadmobile.core.impl.http.UmHttpCall

import okhttp3.Call

/**
 * Simple wrapper to map to OK HTTP library
 */

class UmHttpCallSe
/**
 *
 * @param call the OK Http Call object
 */
(private val call: Call) : UmHttpCall() {

    override fun cancel() {
        call.cancel()
    }
}
